include/glow/Runtime/Executor/Executor.h

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_RUNTIME_EXECUTOR_H
+#define GLOW_RUNTIME_EXECUTOR_H
+
+#include "glow/Graph/Context.h"
+#include "glow/Graph/Graph.h"
+
+#include <map>
+#include <set>
+#include <string>
+
+namespace glow {
+namespace runtime {
```

Comment:
Considered adding this also and think it's a good idea.

Reply:
I like the clear divide it makes. I think it would be the only sub namespace in the codebase. @opti-mix we've discussed namespaces in the past, what are your thoughts?

Reply:
I think using separate namespaces for `runtime` and the like is fine.